src/terms/pairwise.jl

```diff
@@ -0,0 +1,133 @@
+"""
+Pairwise terms: For example Van der Waals energies, such as Lennard—Jones. The potential is
+dependent on the pairwise atomic types.
+"""
+struct PairwisePotential
+    V
+    params
+end
+(P::PairwisePotential)(basis::AbstractBasis) = TermPairwisePotential(P.V, P.params, basis)
+
+struct TermPairwisePotential{TV, Tparams} <:Term
+    V::TV
+    params::Tparams
+    energy::Real
+end
+function TermPairwisePotential(V::TV, params::Tparams, basis::PlaneWaveBasis{T}) where {TV, Tparams, T}
+    TermPairwisePotential(V, params, T(energy_pairwise(basis.model, V, params)))
+end
+
+function ene_ops(term::TermPairwisePotential, basis::PlaneWaveBasis, ψ, occ; kwargs...)
+    (E=term.energy, ops=[NoopOperator(basis, kpt) for kpt in basis.kpoints])
+end
+
+@timing "forces: Pairwise" function compute_forces(term::TermPairwisePotential,
+                                                   basis::PlaneWaveBasis{T}, ψ, occ;
+                                                   kwargs...) where {T}
+    atoms = basis.model.atoms
+    forces_pairwise = zeros(Vec3{T}, sum(length(positions) for (elem, positions) in atoms))
+    energy_pairwise(basis.model, term.V, term.params; forces=forces_pairwise)
+    # translate to the "folded" representation
+    f = [zeros(Vec3{T}, length(positions)) for (type, positions) in atoms]
+    count = 0
+    for i = 1:length(atoms)
+        for j = 1:length(atoms[i][2])
+            count += 1
+            f[i][j] += forces_pairwise[count]
+        end
+    end
+    @assert count == sum(at -> length(at[2]), atoms)
+    f
+end
+
+function energy_pairwise(model::Model, V, params; kwargs...)
+    atom_types = [element for (element, positions) in model.atoms for _ in positions]
+    positions = [pos for (_, positions) in model.atoms for pos in positions]
+
+    energy_pairwise(model.lattice, atom_types, positions, V, params; kwargs...)
+end
+
+"""
+Compute the pairwise interaction energy per unit cell between atomic sites. If `forces` is
+not nothing, minus the derivatives of the energy with respect to `positions` is computed.
+The potential is expected to decrease quickly at infinity, so the reciprocal energy is not
```

Comment:
Put this comment in the terms doc

Reply:
Halfway done. I have put info about potential in terms doc.

Reply:
"The reciprocal energy" doesn't make sense outside of an Ewald context.